_profiles/Sida-Lillian.md

```diff
@@ -8,13 +8,13 @@ hub: CC4D
 languages: "English (Fluent spoken and written), Bari/Kakwa (Fluent spoken and spoken), Arabic (good spoken only), Lugbara (fair spoken only)"
 mail: sidalilian@proton.me
 phone: "+256 (0) 782739162"
-whatsapp: "+256782739162"
+whatsapp: 
 website: 
 telegram: "+256782739162"
-github: Sida1990
+github: sidalillian
 linkedin: 
 twitter: 
-facebook: Lily Savio
+facebook: profile.php?id=100074066215635
 instagram: Sida Lillian
```

Comment:
```suggestion
instagram: 
```

it needs to be the last part of the URL to the profile (everything after the `/` )